cpp/kiss_icp/core/VoxelHashMap.cpp

```diff
@@ -43,42 +43,32 @@ std::vector<Voxel> GetAdjacentVoxels(const Voxel &voxel, int adjacent_voxels = 1
     }
     return voxel_neighborhood;
 }
-std::vector<Eigen::Vector3d> GetPoints(const std::vector<Voxel> &query_voxels,
-                                       const kiss_icp::VoxelHashMap &voxel_map) {
-    std::vector<Eigen::Vector3d> points;
-    points.reserve(query_voxels.size() * static_cast<size_t>(voxel_map.max_points_per_voxel_));
-    std::for_each(query_voxels.cbegin(), query_voxels.cend(), [&](const auto &query) {
-        auto search = voxel_map.map_.find(query);
-        if (search != voxel_map.map_.end()) {
-            const auto &voxel_points = search.value();
-            points.insert(points.end(), voxel_points.cbegin(), voxel_points.cend());
-        }
-    });
-    points.shrink_to_fit();
-    return points;
-}
-
 }  // namespace
 
 namespace kiss_icp {
 
 std::tuple<Eigen::Vector3d, double> VoxelHashMap::GetClosestNeighbor(
-    const Eigen::Vector3d &point) const {
+    const Eigen::Vector3d &query) const {
     // Convert the point to voxel coordinates
-    const auto &voxel = PointToVoxel(point, voxel_size_);
+    const auto &voxel = PointToVoxel(query, voxel_size_);
```

Comment:
Shouldn't this be called `query_voxel` then? to be conistent

Reply:
I have to keep it as ```voxel``` as I want to use ```query_voxel``` into the `for_each`